namespaceFinder.php

```diff
@@ -11,20 +11,35 @@ class namespaceFinder {
     private $wantedNs;
     private $isSafe;
 
+    /**
+     * Resolves the namespace on construction
+     * 
+     * @param string $path the namespace link
+     */
     function __construct($path){
         $this->wantedNs = $this->computeWantedNs($path);
         $this->sanitizeNs();
     }
 
     private function computeWantedNs($path){
         global $ID;
+        global $conf;
+        
+        // Convert all other separators to colons
+        if ($conf['useslash']) $path = str_replace('/', ':', $path);
+        $path = str_replace(';', ':', $path);
+
         $result = '';
         $wantedNS = trim($path);
         if($wantedNS == '') {
             $wantedNS = $this->getCurrentNamespace();
         }
         if( $this->isRelativePath($wantedNS) ) {
             $result = getNS($ID);
+            // normalize initial dots ( ..:..abc -> ..:..:abc )
+            $wantedNS = preg_replace('/^((\.+:)*)(\.+)(?=[^:\.])/', '\1\3:', $wantedNS);
```

Comment:
This is cool, indeed (thanks for your pointer to the doc https://www.dokuwiki.org/namespaces ; I was not aware that this syntax was accepted by Dokuwiki (and that it subsequently makes sense to support it))